src/main.rs

```diff
@@ -55,6 +55,10 @@ struct Opt {
     /// List installed languages
     #[structopt(long)]
     list_languages: bool,
+
```

Comment:
Like I mentioned earlier, to introduce a change like this the default if no flags are specified would have to be the old behavior, or configurable (with the default config still being the old behavior).

The flag should be renamed to `--no-backtrack` or similar.

Reply:
I've done this change. Hope it's ok 😁